package.json

```diff
@@ -64,14 +64,19 @@
     "watch": "yarn test --watch"
   },
   "peerDependencies": {
-    "eslint": "^8.57.0 || ^9.0.0"
+    "eslint": "^8.57.0 || ^9.0.0",
+    "eslint-import-resolver-node": "*"
+  },
+  "peerDependenciesMeta": {
+    "eslint-import-resolver-node": {
+      "optional": true
+    }
   },
   "dependencies": {
     "@typescript-eslint/utils": "^8.32.1",
     "comment-parser": "^1.4.1",
     "debug": "^4.4.1",
-    "eslint-import-context": "^0.1.5",
-    "eslint-import-resolver-node": "^0.3.9",
+    "eslint-import-context": "^0.1.6",
```

Comment:
This is technically a breaking change. Previously, `eslint-import-resolver-node` always existed, but now it does not, and users have to manually install it when needed. Need to land in a major version, where we'd rather just completely ditch those legacy options.